dhnx/optimization/precalc_hydraulic.py

```diff
@@ -19,6 +19,7 @@
 SPDX-License-Identifier: MIT
 """
 
+import logging
 import math
 
```

Comment:
```suggestion
logger = logging.getLogger(__name__)  # Create a logger for this module
```
GitHub seems to allow to only suggest changes close to the edited lines. I would prefer to put this e.g. in line 34 (after all the imports).